hbase-server/src/main/java/org/apache/hadoop/hbase/wal/WALSplitter.java

```diff
@@ -115,7 +119,18 @@
 
   private final String tmpDirName;
 
+  /**
+   * Split WAL directly to hfiles instead of into intermediary 'recovered.edits' files.
+   */
+  public static final String WAL_SPLIT_TO_HFILE = "hbase.wal.split.to.hfile";
+  public static final boolean DEFAULT_WAL_SPLIT_TO_HFILE = false;
+
+  /**
+   * True if we are to run with bounded amount of writers rather than let the count blossom.
+   * Bounded writing tends to have higher throughput.
+   */
   public final static String SPLIT_WRITER_CREATION_BOUNDED = "hbase.split.writer.creation.bounded";
```

Comment:
This defaults to false right?

Reply:
Let me add this.

Reply:
I note that it defaults to 'false' but man, this should be 'true'.  Let me test.